File: caddisfly-app/app/src/androidTest/java/org/akvo/ExternalDeviceSuite.java
package org.akvo;

import org.akvo.caddisfly.test.ECTest;
import org.akvo.caddisfly.test.SensorTest;
import org.akvo.caddisfly.test.SurveySensorTest;
import org.junit.runner.RunWith;
import org.junit.runners.Suite;

// Runs all unit tests.
@RunWith(Suite.class)
@Suite.SuiteClasses({ECTest.class,
        SurveySensorTest.class,
        SensorTest.class})
public class ExternalDeviceSuite {
}

File: caddisfly-app/app/src/androidTest/java/org/akvo/ExternalDeviceSuite.kt
package org.akvo

import org.akvo.caddisfly.test.SensorTest
import org.junit.runners.Suite

// Uncomment @Runwith below to run this suite. This is commented because gradlew runs the test twice
//@RunWith(Suite::class)
@Suite.SuiteClasses(SensorTest::class)
class ExternalDeviceSuite
